x-pack/plugins/upgrade_assistant/server/lib/reindexing/reindex_actions.test.ts

```diff
@@ -64,11 +64,43 @@ describe('ReindexActions', () => {
       });
     });
 
-    it(`replaces -reindexed-v${PREV_MAJOR_VERSION} with -reindexed-v${CURRENT_MAJOR_VERSION}`, async () => {
-      await actions.createReindexOp(`myIndex-reindexed-v${PREV_MAJOR_VERSION}`);
+    it(`prepends reindexed-v${CURRENT_MAJOR_VERSION} to new name, preserving leading period`, async () => {
```

Comment:
Technically this is already covered in parseIndexName, but figure it won't hurt to leave in.